domains/zesk.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "Add some description",
+  "repo": "https://github.com/github-username",
+  "owner": {
+    "username": "github-username",
+    "email": "any@email"
```

Comment:
Need an email address to contact you in case you need to be notified of any changes to is-a-dev. Alternatively, you can keep email as "" and add any of twitter, discord, etc.